nrrd/tests/test_parsing.py

```diff
@@ -1,11 +1,12 @@
 import os
 import sys
 
-sys.path.append(os.path.dirname(os.path.dirname(os.path.dirname(__file__))))
-
 import numpy as np
-from nrrd.tests.util import *
+
 import nrrd
+from nrrd.tests.util import *
+
+sys.path.append(os.path.dirname(os.path.dirname(os.path.dirname(__file__))))
```

Comment:
This shouldn't work. IIRC, the `sys.path` is altered so the `nrrd` module is correctly found.

Not sure how the tests passed?

Reply:
I think it works the same way you are expecting to be honest. I just removed these statements and tests pass the same way so I think they are not needed. I will remove it if that is ok with you